tests/test_manageiq_provider.py

```diff
@@ -86,7 +88,7 @@
 }
 
 GET_RETURN_VALUES = {
-    'openshift_without_hawkular': {
+    'openshift_without_metrics': {
```

Comment:
I wonder how the test passed with the `metrics: true` param. I will fix the test